Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -307,10 +307,6 @@ final class StatusTableViewController: LoopChartsTableViewController {
                 default:
                     break
                 }
-                Task { @MainActor in
-                    refreshContext.update(with: .status)
-                    await reloadData(animated: true)
-                }
```

Comment:
This is the solution for the flicker out partners reported. Note that bolus progress is being updated with `updateBolusProgress`.